.evergreen/.evg.yml

```diff
@@ -263,6 +263,16 @@ functions:
           # DO NOT ECHO WITH XTRACE (which PREPARE_SHELL does)
           PROJECT_DIRECTORY=${PROJECT_DIRECTORY} JDK=${JDK} MONGODB_URI=${gssapi_auth_mongodb_uri} KDC=${gssapi_auth_kdc} REALM=${gssapi_auth_realm} KEYTAB_BASE64=${gssapi_auth_keytab_base64} .evergreen/run-gssapi-auth-test.sh
 
+  "run mmapv1 storage test":
+    - command: shell.exec
+      type: test
+      params:
+        silent: true
+        working_dir: "src"
+        script: |
+          # DO NOT ECHO WITH XTRACE (which PREPARE_SHELL does)
```

Comment:
It's ok for this test to use PREPARE_SHELL.  There is nothing sensitive in it. The command you probably copied this from was one that used passwords that we don't want to leak

Reply:
I have added `${PREPARE_SHELL}` to the script and removed the comment.